src/pyhf/infer/calculators.py

```diff
@@ -543,7 +543,7 @@ def distributions(self, poi_test, track_progress=None):
                     poi_test,
                     sample,
                     self.pdf,
-                    signal_pars,
+                    self.init_pars,
```

Comment:
@lukasheinrich I think these and the `bkg_pars` should instead be set above with something like

```python
signal_pars = init_pars or self.pdf.config.suggested_init()
```

Reply:
that already happends here no

        self.init_pars = init_pars or pdf.config.suggested_init()
 in the init of the calc